sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2131,6 +2263,19 @@ public partial class PredictiveValue
         public System.DateTimeOffset TimeStamp { get { throw null; } }
         public double Value { get { throw null; } }
     }
+    public partial class PrivateLinkScopedResource
```

Comment:
Could we rename this to `MonitorPrivateLinkScopedResource`? The current name is very likely to collide with classes in other RPs

Reply:
Hi @ArcturusZhang. We decided to remove this resource from the SDK. We don't expose it to our customers. 
Could you guide us on how to properly do that?